### PR TITLE
This commit along with a minor change to glue (which I madea PR for) …

### PR DESCRIPTION
**Pull request by scottcoughlin2014**
…allow this to be run in py35

### CODE REVIEW

**Comment by alurban** [comment]
@scottcoughlin2014, does this PR require waiting for your other PR in `glue` to be merged, or is it ready right now?

File: bin/gwdetchar-omega [hunk 1]
```diff
@@ -27,7 +27,10 @@ import re
 import sys
 import ast
 import warnings
-from StringIO import StringIO
+try:
+    from StringIO import StringIO
+except ImportError:
+    from io import StringIO
```
Comment (duncanmmacleod) [inline bin/gwdetchar-omega]: @scottcoughlin2014 (@alurban) can we swap the ordering of this to make it python3-first?

Comment (scottcoughlin2014) [reply bin/gwdetchar-omega]: Done

**Comment by scottcoughlin2014** [comment]
@alurban Sure this can be merged without the `glue` merge. all it means is that it will still not work in python 3 until that is merged but not that this will all of a sudden stop working or whatever.

**Comment by alurban** [comment]
Ok, I will merge. Thanks @scottcoughlin2014 for putting this together and @duncanmmacleod for reviewing

**Pull request merged**